drivers/cache/cache_handlers.c

```diff
@@ -7,30 +7,32 @@
 #include <zephyr/cache.h>
 #include <zephyr/syscall_handler.h>
 
-static inline int z_vrfy_sys_cache_data_all(int op)
+static inline int z_vrfy_sys_cache_data_flush_all(void)
 {
-	return z_impl_sys_cache_data_all(op);
+	return z_impl_sys_cache_data_flush_all();
```

Comment:
This is kind of an odd thing. To flush a range of memory the user mode thread will have permissions to that range, but here to flush all memory no permissions are checked.

Are there potential negative implications of a user mode thread spamming cache flushes?

Reply:
Yeah, indeed this seems odd. About the potential negative implications of spamming flushes should be none besides performance degradation. At this point I'll probably remove this as well from the syscalls list and just leave the `_range` operations for which we can check the range.

Reply:
done